clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -161,8 +164,7 @@ public class AzureComputeClient extends AzureBaseClient {
   }
 
   ServiceResponse<Void> destroyServerGroup(String resourceGroupName, String serverGroupName) {
-    VirtualMachineScaleSetsOperations ops = getAzureOps(
-      client.&getVirtualMachineScaleSetsOperations, "Get operations object", "Failed to get operation object") as VirtualMachineScaleSetsOperations
+    VirtualMachineScaleSetsOperations ops = client.getVirtualMachineScaleSetsOperations()
```

Comment:
Did the Azure JSDK team confirm that getting the operation object will not result into a REST call to Azure?